conf/janus.transport.mqtt.jcfg.sample

```diff
@@ -7,6 +7,7 @@ general: {
 	url = "tcp://localhost:1883"		# The connection URL of the MQTT broker: if you want
 										# to use SSL, make sure you type ssl:// instead of tcp://,
 										# and that you configure the SSL settings below
+	#mqtt_version = "3.1.1"     			# Protocol version. Available values: 3.1, 3.1.1 (default), 5.
```

Comment:
It looks like it's still spaces between the value and the comment. Can you fix that too?